### ISSUE TITLE
Cannot run the tar.gz file in linux

### ISSUE THREAD

**Issue by singhhrpreet** [prasmussen/gdrive#580]
Hi,

I downloaded the pre-built binary. This exact link 👇🏼 
(https://github.com/prasmussen/gdrive/releases/download/2.1.1/gdrive_2.1.1_linux_amd64.tar.gz)

But, when I extract and try to run the executable, it shows

./gdrive

it says

`No such file or directory`

**Comment by kami93** [comment]
FIlenames for i386 and amd64 are swapped. Download https://github.com/prasmussen/gdrive/releases/download/2.1.1/gdrive_2.1.1_linux_386.tar.gz instead.

**Comment by TonalidadeHidrica** [comment]
@prasmussen This is causing problem to quite a many people.  Could you fix the download link, or at least pin this issue?

**Comment by metaphore** [comment]
I don't think that the binaries are actually swapped. Because `file` command against the 386 version of `gdrive` clearly states that it is a `ELF 32-bit LSB executable`. And it returns `ELF 64-bit LSB executable, x86-64` for a amd64 archive version.

So the 386 version works in general case, because there's a 32-bit instruction interpreter...

The true problem is actually a missing dependency (as stated in the #618 issue).
You need to have `musl` lib installed on the system in order to get a working 64-it executable.

For Ubuntu based distros simply execute
```
sudo apt install musl -y
```

**Comment by klarkc** [comment]
This `musl` dependency can't be embed in the binary?
